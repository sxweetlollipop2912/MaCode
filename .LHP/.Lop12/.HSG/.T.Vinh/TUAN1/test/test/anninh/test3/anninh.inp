8 10
4 82
7 94
12 20
25 42
33 75
39 43
69 10
76 62
60 41
2 10
85 54
32 55
78 96
59 5
15 29
71 82
76 19
21 85
