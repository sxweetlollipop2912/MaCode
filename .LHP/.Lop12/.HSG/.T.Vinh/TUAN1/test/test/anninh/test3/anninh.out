21
10
9
12
34
10
9
20
9
12
